doc(pre-commit): setup instructions to docs (#8465)
internetarchive/openlibrary: CONTRIBUTING.md

npm run lint-fix
```

## Quality code

Be confident in changing files you can check the quality (linter) with [pre-commit](https://pre-commit.com/index.html)

> Pre-commit is used to inspect the snapshot that is about to be committed, to see if you missed anything, or to review what you need to inspect in the code. You can see the actions descriptions in the **pre-commit-config.yml** file.

Reviewer: ```suggestion
Pre-commit is used to inspect the snapshot that is about to be committed, to see if there are any syntax errors, typos, or a handful of other common issues. You can see the actions descriptions in [pre-commit-config.yml](https://github.com/internetarchive/openlibrary/blob/master/.pre-commit-config.yaml).
```

> The pre-commit is launched for PR validations, it is important to install the pre-commit locally to avoid unverified code being pushed.

Reviewer: ```suggestion
The pre-commit is automatically run against open PRs. Install the pre-commit locally to avoid waiting for the PR checks to run in the cloud.
```

### Installation

````
pip install pre-commit

>  or
> $ brew install pre-commit (Homebrew)

Reviewer: ```suggestion
# or on mac you can run
brew install pre-commit
```

> 
> $ pre-commit install
> ````
> 

Reviewer: ```suggestion
````

Then run `pre-commit install` from the root of the openlibrary directory to set up the git hook scripts
```

Author: I don't consider this comment :)

> After executing the last command, when you normally run ``git commit``, pre-commit will also perform its checks.

Reviewer: ```suggestion
After executing the last command, when you normally run `git commit`, pre-commit will also perform its checks.
```
